MentalHospital.Tests/Tests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Xunit;
+using Moq;
+using AutoMapper;
+using Shouldly;
+using MentalHospital.BLL.Mapper;
+using MentalHospital.BLL.Models;
+using MentalHospital.BLL.Services;
+using MentalHospital.DAL.Interfaces;
+using MentalHospital.DAL.Entities;
+
+
+namespace MentalHospital.Tests
+{
+    public class Tests
+    {
+        private readonly IMapper mapper = new Mapper(new MapperConfiguration(
+            cfg => cfg.AddProfile<MappingProfile>()));
+        private Mock<IPatientRepository> stubPatientRepository = new Mock<IPatientRepository>();
+        private readonly string notAssignedMsg = "Method returned value is not assingable with expected type";
+
+        [Fact]
+        public void Creaate_InputPatientModel_ReturnSameType()
+        {
+            var patientModel = InitialPatientModel();
+            stubPatientRepository.Setup(rep => rep.Create(It.IsAny<Patient>()))
+                .Returns<Task<Patient>>(patient => patient);
+            var patientService = new PatientService(stubPatientRepository.Object, mapper);
+
+            var result = patientService.Create(patientModel);
```

Comment:
Test method also can be async and you can use async/await

Also read pls about - how to invoke async method in sync context